test/test_hyperparameters.py

```diff
@@ -239,6 +239,18 @@ def test_normalfloat(self):
         self.assertNotEqual(f1, f2)
         self.assertNotEqual(f1, "UniformFloat")
 
+        with pytest.raises(ValueError):
+            f6 = NormalFloatHyperparameter("param", 5, 10, lower=0.1, upper=0.1, default_value=5.0, q=0.1, log=True)
+
+        f6 = NormalFloatHyperparameter("param", 5, 10, lower=0.1, upper=10, default_value=5.0, q=0.1, log=True)
+        f6_ = NormalFloatHyperparameter("param", 5, 10, lower=0.1, upper=10, default_value=5.0, q=0.1, log=True)
+        self.assertEqual(f6, f6_)
+        self.assertEqual(
+            "param, Type: NormalFloat, Mu: 5.0 Sigma: 10.0, Range: [0.1, 10.0], Default: 5.0, on log-scale, Q: 0.1", str(f6))
+
+        self.assertNotEqual(f1, f2)
```

Comment:
This is new duplicated.

Reply:
I am sorry, I don't see the duplicated? I followed the previous test cases with `f4, f4_` which was already there.